cmp/grid/impl/MultiFieldRenderer.js

```diff
@@ -0,0 +1,105 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+
+import {Component} from 'react';
+import {PropTypes as PT} from 'prop-types';
+import {elemFactory, HoistComponent} from '@xh/hoist/core';
+import {Record} from '@xh/hoist/data';
+import {GridModel} from '@xh/hoist/cmp/grid';
+import {vbox, hbox, span} from '@xh/hoist/cmp/layout';
+import {throwIf} from '@xh/hoist/utils/js';
+
+import {SubField} from './SubField';
+
+/**
+ * Renders a collection of additional sub fields in a row beneath the main column field.
+ * Typically not used directly - instead created by using the MultiFieldRenderer elementRendererFn.
+ * @private
+ */
+@HoistComponent
+class MultiFieldRenderer extends Component {
+
+    static propTypes = {
+        /** Primary value to render */
+        value: PT.oneOfType([PT.string, PT.number, PT.bool]).isRequired,
+        /** The data Record to render. */
+        record: PT.instanceOf(Record).isRequired,
+        /** Reference to the Column */
```

Comment:
I might just pass the column renderer metadata in as a single bundle, and read the  multiFieldRendererCfg right off the column from inside this component.  Makes the whole interface simpler -- the renderer truly just becomes a pass through.  Then you also have all the metadata available for passing on to the actual renderers on 91, without having to reconstruct -- less fragile